src/functions/components/FunctionCard.tsx

```diff
@@ -38,16 +39,15 @@ const FunctionCard: FC<Props> = ({id, name}) => {
       direction={FlexDirection.Row}
       contextMenu={<FunctionContextMenu id={id} name={name} />}
     >
-      <LastRunFunctionStatus lastRunError={undefined} lastRunStatus="success" />
       <FlexBox
         alignItems={AlignItems.FlexStart}
         direction={FlexDirection.Column}
         margin={ComponentSize.Medium}
       >
         <ResourceCard.Name name={name} onClick={handleClick} />
         <ResourceCard.Meta>
-          <>Last triggered 3 minutes ago</>
-          <>Trigger URL: http://moo.influxdata.com/yerfunctionisawesome</>
+          <>{description && description}</>
```

Comment:
can't you just put... description?

Reply:
I can.. I had something else there before.. :)